src/core/public/application/types.ts

```diff
@@ -826,9 +767,3 @@ export interface InternalApplicationStart extends Omit<ApplicationStart, 'regist
    */
   history: History<unknown> | undefined;
```

Comment:
Can this still be undefined? Shouldn't it always be present now since we no longer have a legacy mode?